tiledb/common/interval/CMakeLists.txt

```diff
@@ -25,7 +25,9 @@
 # THE SOFTWARE.
 #
 
-cmake_policy(SET CMP0076 NEW)  # CMake 3.13 - converts relative paths to absolute
+if (POLICY CMP0076) # CMP0076 unknown to vs2017 cmake version 3.12.18081601-MSVC_2
+  cmake_policy(SET CMP0076 NEW)  # CMake 3.13 - converts relative paths to absolute
+endif()
```

Comment:
Only thing I'd change is to move this condition to a common .cmake file at the top, since this kind of thing isn't going to be the only location we'll need this policy set, nor is it the only kind of commonality we'll want with unit-based development.

And since I would change this, I'll put up a PR for it later.